feature-libs/customer-ticketing/core/facade/customer-ticketing.service.ts

```diff
@@ -120,14 +122,19 @@ export class CustomerTicketingService implements CustomerTicketingFacade {
   protected uploadAttachmentCommand: Command<{
     file: File | null;
```

Comment:
` file: File | null; `. shouldn't be null, right?

Reply:
Fixed https://github.com/SAP/spartacus/pull/16383/commits/67f773742f3c238689ded8a66069c9fafa878ebc